board/safety/safety_volkswagen.h

```diff
@@ -83,6 +143,16 @@ static int volkswagen_mqb_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
       volkswagen_gas_prev = gas;
     }
 
+    // Exit controls on rising edge of brake press
+    // Signal: ESP_05.ESP_Fahrer_bremst
+    if ((bus == 0) && (addr == MSG_ESP_05)) {
+      bool brake_pressed = (GET_BYTE(to_push, 3) & 0x4) >> 2;
+      if (brake_pressed && (!(volkswagen_brake_pressed_prev) || volkswagen_moving)) {
```

Comment:
this should be:
- rising edge of brake press always cancels
- brake pressed with moving car also cancels

The case not covered here is: car is stopped, system is engaged and user presses the brake.

Reply:
edit: my bad, did not read it properly. It's fine